models/welcome.js

```diff
@@ -0,0 +1,20 @@
+var xtend = Object.assign
+
+module.exports = welcomeModel
+
+function welcomeModel (state, bus) {
+  state.welcome = xtend({
+    show: false
+  }, state.welcome)
+
+  bus.on('repos loaded', function () {
```

Comment:
This is not a blocker btw, perhaps something to discuss either over text or face? feel we can figure it out in 5mins when talking haha